src/App.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import {Header} from '@Modules/Header';
```

Comment:
I would suggest to use start folder names from lower case letter.

```suggestion
import {Header} from '@Modules/Header';
```